src/sentry/static/sentry/app/components/events/eventEntries.tsx

```diff
@@ -45,39 +65,194 @@ type Props = {
   event: Event;
   project: Project;
   location: Location;
-
+  api: Client;
   group?: Group;
   className?: string;
 } & typeof defaultProps;
 
-class EventEntries extends React.Component<Props> {
+type State = {
+  isLoading: boolean;
+  proGuardErrors: ProGuardErrors;
+};
+
+class EventEntries extends React.Component<Props, State> {
   static defaultProps = defaultProps;
 
+  state: State = {
+    isLoading: true,
+    proGuardErrors: [],
+  };
+
   componentDidMount() {
+    this.checkProGuardError();
+    this.recordIssueError();
+  }
+
+  shouldComponentUpdate(nextProps: Props, nextState: State) {
+    const {event, showExampleCommit} = this.props;
+
+    return (
+      (event && nextProps.event && event.id !== nextProps.event.id) ||
+      showExampleCommit !== nextProps.showExampleCommit ||
+      nextState.isLoading !== this.state.isLoading
+    );
+  }
+
+  async fetchDebugFile(query: string): Promise<Array<DebugFile>> {
+    const {api, organization, project} = this.props;
+    try {
+      const debugFiles = await api.requestPromise(
+        `/projects/${organization.slug}/${project.slug}/files/dsyms/`,
+        {
+          method: 'GET',
+          query: {
+            query,
+            file_formats: 'proguard',
+          },
+        }
+      );
+      return debugFiles;
+    } catch (error) {
+      Sentry.captureException(error);
```

Comment:
Is this something you want to be alerted on?

Reply:
Yes, I think it's important to know when this request failed so we can check what happened

Reply:
You might get a bunch of request aborted errors, but this can be removed if it gets too noisy.